codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/protocol/aws/AwsJson10ProtocolGenerator.java

```diff
@@ -49,14 +54,22 @@ public void generateRequestSerializers(GenerationContext context) {
         var writer = context.getWriter().get();
         var model = context.getModel();
         var ops = model.getOperationShapes();
+        Set<Shape> shared = new java.util.HashSet<>(Set.of());
         for (var op : ops) {
-            requestSerializerCode(op, writer);
+            var middle = new SerializeMiddleware(this, context, op, writer);
+            writer.write(middle.generateOperationStubs());
+            writer.write("\n");
+            Set<Shape> shapes = getShapesToSerde(context.getModel(), context.getModel().expectShape(
+                    op.getInputShape()));
+            shared.addAll(shapes);
         }
+        var generator = new JsonSerializerGenerator(context.getModel(), context.getSymbolProvider());
+        writer.write(generator.generate(shared));
     }
 
     @Override
     public void generateResponseDeserializers(GenerationContext context) {
-        var writer = context.getWriter().get();
+        GoWriter writer = context.getWriter().get();
```

Comment:
No reason to change, things like this clog the diff in the PR, making the change harder to review.

Reply:
This is marked as resolved, but the type was still changed. What Luc means is "move it back to how it originally was" (i.e. `var writer`) so it doesn't show up in the diff and we can focus only on the things that have actually changed

Reply:
Ah got it, sorry I thought that changing it back now would cause a new diff deletion/insert highlight. Changed it back to the original var.